store_app/views/queries.py

```diff
@@ -0,0 +1,88 @@
+from flask import make_response, jsonify, request
+from tables.models import Movie, User, UserPurchase
+from app import app
+from business import MovieCRUD
+
+
+@app.route('/movies/total-count', methods=['GET'])
+def get_count_entries():
+    try:
+        count = Movie.query.count()
+        return make_response(jsonify(count))
+    except Exception as e:
+        print(f'error: {str(e)}')
+        return make_response(jsonify(error=str(e)), 500)
+
+
+@app.route('/movies/title/<string:title>', methods=['GET'])
+def get_movies_by_title(title):
+    try:
+        movies = Movie.query.filter(Movie.title.like(f'%{title}%')).all()
+        return make_response(jsonify([m.serialize() for m in movies]), 200)
+    except Exception as e:
+        print(f'error: {str(e)}')
+        return make_response(jsonify(error=str(e)), 500)
+
+
+@app.route('/movies/price-range', methods=['GET'])
+def get_movies_in_price_range():
+    try:
+        max_price = request.args.get('max_price')
+        min_price = request.args.get('min_price')
```

Comment:
You could handle missing query parameters better, e.g. telling the user that these are mandatory.